stream-chat-android-state/src/main/java/io/getstream/chat/android/state/plugin/logic/channel/internal/ChannelStateLogic.kt

```diff
@@ -432,6 +437,7 @@ internal class ChannelStateLogic(
         // this means that if the offline sync went out of sync things go wrong
         upsertMembers(c.members)
         upsertWatchers(c.watchers, c.watcherCount)
+        upsertMessages(c.messages, false)
```

Comment:
This seems to be a **main** fix. Am I right?

Reply:
Yes, that's correct.